tutorevent_bus_redis/plugin.py

```diff
@@ -13,13 +13,107 @@
 ########################################
 # CONFIGURATION
 ########################################
+# FIXME: Update this to a saner config structure less likely to break, and able
+# to activate and deactivate individual events more easily.
+PRODUCER_CONFIG = """{
```

Comment:
why aren't you using a proper dictionary?

Reply:
Just hadn't gotten to it yet, I'm still trying to figure out the best way to handle this config from an operator perspective.